packages/core/src/Provider/Provider.ts

```diff
@@ -8,6 +8,7 @@ export type HealthCheckResponse = {
     ledgerTip?: Tip;
     networkSync?: Percent;
   };
+  projectedTip?: PartialBlockHeader;
```

Comment:
I chosen `PartialBlockHeader` (which is actually an alias for `Cardano.Tip`), to avoid ambiguity with `import { Tip } from '@cardano-ogmios/schema';`.
I'll try with an explicit `Cardano.` before `Tip`.

Reply:
Yes, that would be better semantically

Reply:
ea59ed748 should solve this